src/UNL/UCBCN/Manager/Calendar.php

```diff
@@ -38,4 +43,28 @@ public function getCategorizedEvents() {
         return $categories;
     }
 
+    private function archiveEvents() {
+        # find all posted (upcoming) events on the calendar
+        $events = $this->calendar->getEvents('posted');
+
+        # check each event to see if it has passed
+        foreach ($events as $event) {
+            # we will consider it passed if one datetime is in the past
+            # (this is how the previous system did it)
```

Comment:
I think this has caused confusion in the past, and will be especially confusing given the new UI.  It should be the other way around, only mark it as archived if ALL datetimes are in the past.